metrics/beaconstate.go

```diff
@@ -110,15 +111,15 @@ func (p *BeaconState) Run() {
 		// TODO: Sync committee
 		// TODO:
 		nOfSlashedValidators := 0
-		for _, val := range currentBeaconState.Altair.Validators {
+		for _, val := range currentBeaconState.Bellatrix.Validators {
```

Comment:
Would recommend having a look at #18. The way i fixed it is by supporting both Bellatrix and Altair. No that the software has to also work in mainnet, where the merge didn't happen yet.